install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,410 @@
+
+. Create the installation playbook, *_/bastion_installation/create-bastion-machine-playbook.yaml_*, and update its parameters:
++
+[source,yml]
+----
+---
+- name: Create a bastion machine
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://_Manager_FQDN_/ovirt-engine/api
+    engine_user: <admin@internal>
+    engine_password: "{{ engine_password }}"
+    engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+    qcow_url: <1>
+    template_cluster: Default
+    template_name: rhelguest7
+    template_memory: 4GiB
+    template_cpu: 2
+    wait_for_ip: true
+    debug_vm_create: false
 
-To integrate {product-title} with Red Hat Virtualization, take the following
-actions as part of your
-xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+    vms:
+    - name: rhel-bastion
+      cluster: Default
+      profile:
+        cores: 2
+        template: "{{ template_name }}"
+        root_password: "{{ root_password }}"
+        ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocpadmin.pub') }}"
+        state: running
+      cloud_init:
+        custom_script: |
+          rh_subscription:
+            username: "{{ rhsub_user }}"
+            password: "{{ rhsub_pass }}"
+            auto-attach: true
+            disable-repo: ['*']
+            # 'rhel-7-server-rhv-4.2-manager-rpms' supports RHV 4.2 and 4.3
+            enable-repo: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ansible-2.7-rpms', 'rhel-7-server-ose-3.11-rpms', 'rhel-7-server-supplementary-rpms', 'rhel-7-server-rhv-4.2-manager-rpms']
+          packages:
+            - ansible
+            - ovirt-ansible-roles
+            - openshift-ansible
+            - python-ovirt-engine-sdk4
+  tasks:
+    - user
+        name: root
 
-. To provide high availability in case of the loss of one hypervisor host,
-add each class of instance to a negative affinity group. See
-link:https://ovirt.org/develop/release-management/features/sla/vm-affinity/[VM Affinity].
+  roles:
+    - oVirt.image-template
+    - oVirt.vm-infra
 
-. To ensure that the {product-title} environment meets the
-xref:../install/prerequisites.adoc#hardware[minimum hardware requirements],
-create templates for virtual machines that use the following resources:
+- name: post installation tasks on the bastion machine
+  hosts: rhel-bastion
+  tasks:
+    - name: create ovirt-engine pki dir
+      file:
+        state: directory
+        dest: /etc/pki/ovirt-engine/
+    - name: Copy the engine ca cert to the bastion machine
+      copy:
+        src: "{{ engine_cafile }}"
+        dest: "{{ engine_cafile }}"
+    - name: Copy the secured vars to the bastion machine
+      copy:
+        src: secure_vars.yaml
+        dest: secure_vars.yaml
+        decrypt: false
+    - name: copy the ocpadmin keypair to the bastion machine
+      copy:
+        src: "{{ item }}"
+        dest: "{{ item }}"
```

Comment:
@rgolangh This will fail because path `/root/.ssh` does not exist on the bastion. As a workaround for my testing, I had to add following task before this one:
```
- file:
    state: directory
    path: /root/.ssh
```